devicetypes/smartthings/zigbee-multi-switch.src/zigbee-multi-switch.groovy

```diff
@@ -107,6 +107,20 @@ def parse(String description) {
 	Map eventMap = zigbee.getEvent(description)
 	Map eventDescMap = zigbee.parseDescriptionAsMap(description)
 
+	if ( device.getDataValue("deviceJoinName") == "eZEX Switch 1") {
+		if (eventMap && eventMap?.name == "switch" && (eventMap?.value == "on" || eventMap?.value == "off")) {
```

Comment:
You shouldn't have to check for `on` or `off`. You won't get a switch event from `zigbee.getEvent` without either of those values.

Reply:
Okay, I will delete this unnecessary condition.